src/P3Scheme.jl

```diff
@@ -0,0 +1,149 @@
+"""
+Predicted particle properties scheme (P3) for ice, which includes:
+ - m(D) regime
+ - A(D) regime
+ - particle fall speed regime
+ - add more (source/sink) as we go! (TODO)
+"""
+module P3Scheme
+
+import CloudMicrophysics as CM
+import Thermodynamics as TD
+import NonlinearSolve as NLS
+
+const CT = CM.CommonTypes
+const CO = CM.Common
+const CMP = CM.Parameters
+const APS = CMP.AbstractCloudMicrophysicsParameters
+
+const FT = Float64
```

Comment:
It's ok to keep it for now, but in general we want to write code that will work for different floating-point precision. So once we are nearly done with the PR, we should move them to CLIMAParameters repository.

Also, do we need all of those parameters? I think that the density of ice is already defined in CLIMAParameters.

Reply:
Sounds good! I'll get rid of all the parameters that aren't relevant to the m(D) regime. (Some of those are for A(D) and V(D).) Also, I see that in CliMAParameters we have "density_ice_water": I'll try to use that.

Reply:
Actually also saw that in CloudMicrophysics Parameters we have `\rho_cloud_ice`, but I'm having trouble using that. I'll ask for help next week if I still can't figure it out.

Reply:
Are the problems in the source code or when you try running in tests/documentation?

In the source code the general pattern should be that you pass in the parameter set as one of the arguments to your function. So something like this:

```
import ..Parameters as CMP
const APS = CMP.AbstractCloudMicrophysicsParameters
function foo(prs::APS, ρ::FT,) where {FT <: Real}
    _ρ_cloud_liq::FT = CMP.ρ_cloud_liq(prs)
    # more code
    return 0
end
```

Inside tests/examples you actually have to create that parameter set, which unfortunately requires some more lines of code. Maybe something like this:

```
import CLIMAParameters as CP
include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
FT = Float64
toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
prs = cloud_microphysics_parameters(toml_dict)
foo(prs, FT(1))
```

Reply:
Ohh ok that makes sense, thanks! I think I was trying to create the parameter set in my source code, which was causing problems. I got it to work for docs, which is actually where we need it to work! I'll make it so that the parameter is passed as an argument.